src/content/en/updates/2019/02/using-twa.md

```diff
@@ -442,6 +442,119 @@ adb logcat | grep -e OriginVerifier -e digital_asset_links
 With the upload APK generated, you can now [upload the app to the Play
 Store](https://developer.android.com/studio/publish/upload-bundle).
 
+## Adding a Splash Screen {: #making-a-splash }
+
+Starting on **Chrome 75**, Trusted Web Activities are have support for Splash Screens.
+The Splash Screen can be added by adding a few new image files and configurations to the
```

Comment:
"can be added by adding" - a bit awkward. Perhaps "can be set up by adding"?

Reply:
fixed